app/src/main/java/com/codepath/apps/critterfinder/models/PetModel.java

```diff
@@ -18,6 +18,16 @@
     String age;
     String size;
     String description;
+    String contactName;
+    String contactPhone;
+    String contactEmail;
+    String contactState;
+    String contactCity;
+    String contactZip;
+    String contactAdress1;
+    String contactAdress2;
+
```

Comment:
Good  Point Scott M. I did it it s because we  did not have to call all the pets for a specific shelter. In a relation 1 to many ( shelter --> Pets)

Cool


Sent from my iPhone

> On Mar 8, 2016, at 9:32 PM, Scott MacGregor <notifications@github.com> wrote:
> 
> In app/src/main/java/com/codepath/apps/critterfinder/models/PetModel.java:
> 
> > @@ -18,6 +18,16 @@
> >      String age;
> >      String size;
> >      String description;
> > +    String contactName;
> > +    String contactPhone;
> > +    String contactEmail;
> > +    String contactState;
> > +    String contactCity;
> > +    String contactZip;
> > +    String contactAdress1;
> > +    String contactAdress2;
> > +
> just a suggestion for future learning - if we were building a production ready app we'd probably not store the shelter information directly on the pet model because it isn't a pet. Instead we'd create a separate model class called "Shelter". That model would have the properties you added here. Our pet model would then have a Shelter property on it like:
> 
> i.e.
> 
> Shelter shelter;
> It helps with data encapsulation between our models.
> 
> —
> Reply to this email directly or view it on GitHub.
>